addons/mail/static/src/models/follow_button_view.js

```diff
@@ -6,6 +6,20 @@ import { one } from '@mail/model/model_field';
 registerModel({
     name: 'FollowButtonView',
     identifyingFields: [['chatterOwner']],
+    recordMethods: {
+        /**
+         * @param {MouseEvent} ev
+         */
+        onClickUnfollow(ev) {
+            if (!this.exists()) {
+                return;
+            }
+            if (!this.chatterOwner.thread) {
+                return;
+            }
+            this.chatterOwner.thread.onClickUnfollow(ev);
```

Comment:
Better call `Thread/unfollow` directly and remove `onClickUnfollow` from `Thread`.

Reply:
Changes done.